Scripts/OneDrive-Client-GPO-Configuration/readme.md

```diff
@@ -0,0 +1,35 @@
+# Title of the Script (SEO friendly) #
```

Comment:
One more small thing to fix... as this will be the title of the readme, so let's ensure that it's SEO friendly title... "OneDrive Client Group Policy (GPO) configuration" or something similar. Missed this in last check. This will help for people to find it as it's used by Google and Bing to identify the file.

Reply:
Alright, I _think_ this should be good to go. Let me know if anything else needs a correction.